### PR TITLE
Increase the size of collect switch

### PR DESCRIPTION
**Pull request by josevalim**
Screenshot 2023-09-25 at 19 08 18

I have also made the spacing at the top even.

### CODE REVIEW

File: lib/assets/data_transform_cell/main.js [hunk 1]
```diff
@@ -124,8 +124,8 @@ export async function init(ctx, payload) {
         v-bind="$attrs"
         v-bind:class="inputClass"
       >
-      <div class="validation-wrapper">
-        <span class="tooltip right validation-message" :data-tooltip="message" v-if="message">
+      <div class="validation-wrapper" v-if="message">
+        <span class="tooltip right validation-message" :data-tooltip="message">
```
Comment (josevalim) [inline lib/assets/data_transform_cell/main.js:128]: @cristineguadelupe I have no idea if this change is OK. Can you please confirm?

**Comment by josevalim** [comment]
I also pushed a fix to validation wrapper:

Screenshot 2023-09-25 at 19 12 21

**Comment by josevalim** [comment]
I also aligned the icons on the right (compare to previous image). Same distance from top and bottom and align center:
Screenshot 2023-09-25 at 19 23 58

File: lib/assets/data_transform_cell/main.js (hunk 1 above)
Comment (cristineguadelupe) [reply lib/assets/data_transform_cell/main.js:128]: Yes, it is!

**Review by cristineguadelupe — APPROVED** [review]

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**